IRelectra.h

```diff
@@ -61,22 +61,22 @@ class MarkSpaceArray
     // in the Manchester code.
     MarkSpaceArray(uint16_t unitLengthInUsec);
 
-    // Add a number of time units with mark.
+    // Add a number of time units with mark should be appended.
     void addMark(uint16_t units);
 
-    // Add a number of time units with space.
+    // Add a number of time units with space should be appended.
     void addSpace(uint16_t units);
-    
-    // Encodes the bit with IEEE 802.3 Manchester coding and adds it to the array
+
+    // Encodes the bit with IEEE 802.3 Manchester coding and them to the array
     // A zero bit is one unit MARK and one unit SPACE
-    // a one bit is one unit SPACE and one unit MARK
+    // A one bit is one unit SPACE and one unit MARK
     void addBitWithManchesterCode(uint8_t bit);
-    
-    // Encodes a given number of bits from the given number bit by bit with 
-    // IEEE 802.3 Manchester coding and adds it to the array. MSB first.
+
+    // Encodes a given number of bits from the given number bit by bit with
+    // IEEE 802.3 Manchester coding and add it to the array. MSB first.
```

Comment:
This is incorrect IMO. The original is fine, note the doc begin with `encodes` so `adds` is OK. All docs should use this style.